Traceback (most recent call last):
  File "/home/SystemGNP/OperacionCND/CNDWeb/CrmCND/src/app.py", line 10, in <module>
    import flask_login
  File "/usr/local/lib/python3.8/site-packages/flask_login/__init__.py", line 12, in <module>
    from .login_manager import LoginManager
  File "/usr/local/lib/python3.8/site-packages/flask_login/login_manager.py", line 33, in <module>
    from .utils import _create_identifier
  File "/usr/local/lib/python3.8/site-packages/flask_login/utils.py", line 14, in <module>
    from werkzeug.urls import url_decode
ImportError: cannot import name 'url_decode' from 'werkzeug.urls' (/usr/local/lib/python3.8/site-packages/werkzeug/urls.py)
